src/js/components/FocusedContainer.js

```diff
@@ -43,19 +44,48 @@ export const FocusedContainer = ({
     };
   }, [bodyOverflowStyle, hidden, trapFocus, restrictScroll]);
 
+  const trapFocusHandler = (event) => {
```

Comment:
This needs to be memoized with useCallback or something like that so it doesn't change on every render